.. _Exceptions:

pycatia.exception_handling.exceptions
=====================================

.. automodule:: pycatia.exception_handling.exceptions
    :members: